src/main/external-resources/renderers/Samsung-BDC6800.conf

```diff
@@ -32,3 +32,4 @@ Supported = f:wmv      v:vc1|wmv            a:wma                    m:video/x-m
 # Supported audio formats:
 Supported = f:mp3   m:audio/mpeg
 Supported = f:wma   m:audio/x-ms-wma
+MediaInfo = false
```

Comment:
This is interesting since I think we have found a bug with this configuration - it has the `Supported` lines without the `MediaInfo = true`. In this case let's leave it, so can you please remove this line?